pom.xml

```diff
@@ -1122,6 +1132,7 @@
 		<junitVersion>5.8.2</junitVersion>
 		<mockitoVersion>3.12.4</mockitoVersion>
 		<hamcrestVersion>2.2</hamcrestVersion>
+		<owaspCsrfGuardVersion>4.0.0</owaspCsrfGuardVersion>
```

Comment:
Is there any reason why you are not using the latest release of this module?

Reply:
there readme on giithub showed `4.0.0` so I thought `4.1.1` was still in development or something like that  
Am gonna try `4.0.0` is failing the security check.  Maybe `4.0.0`won't